crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -265,6 +265,11 @@ mod ffi {
     }
 }
 
+pub use ffi::QFontCapitalization;
```

Comment:
why not `pub use ffi::{QFontCapitalization, QFontHintingPreference, QFontSpacingType, QFontStyle};` ?

Reply:
Yep why not :)
I will change it.